network-api/networkapi/wagtailpages/templates/wagtailpages/index_page.html

```diff
@@ -59,7 +62,7 @@ <h3 class="tw-body result-info">
     {% endif %}  {# End of IF for filtered page content _after_ starting a new row. #}
       {% block subcontent %}
         {% include "./fragments/entry_cards.html" %}
-      {% endblock %}
+    {% endblock %}
```

Comment:
**Suggestion** The above two lines should be out-dented too.

Reply:
Good catch! Updated